--- /data/data/com.termux/files/home/docs/rbnacl-libsodium-1.0.13/vendor/libsodium/configure
+++ /data/data/com.termux/files/home/rbnacl-libsodium-1.0.13/vendor/libsodium/configure
@@ -2330,8 +2330,8 @@
 ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS >&5'
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
-
-ac_aux_dir=
+SHELL="/data/data/com.termux/files/usr/bin/sh"
+ac_aux_dir="./build-aux"
 for ac_dir in build-aux "$srcdir"/build-aux; do
   if test -f "$ac_dir/install-sh"; then
     ac_aux_dir=$ac_dir
